authentificator.rb

```diff
@@ -0,0 +1,34 @@
+require 'dotenv'
+
+class Authentificator
+  SCOPES        = ['https://www.googleapis.com/auth/userinfo.email'].join(' ')
+  SITE          = 'https://accounts.google.com'.freeze
+  AUTHORIZE_URL = '/o/oauth2/auth'.freeze
+  TOKEN_URL     = '/o/oauth2/token'.freeze
+  APP_BASE_URL  = 'http://localhost:4567'.freeze
```

Comment:
you're using localhost in callback url
I'm sure it will not work when you deploy it to heroku. Right?
you will need to load APP_BASE_URL from ENV variable and do setup config at heroku with a proper value for that var.